src/table/body-cell/index.tsx

```diff
@@ -9,6 +9,7 @@ import { TableTdElement, TableTdElementProps } from './td-element';
 import { InlineEditor } from './inline-editor';
```

Comment:
When I click on the success icon after a successful edit, the icon disappears. That seems like a strange interaction?

Reply:
Good catch (both issues)! 
This actually happened because the button's onBlur event got fired which removes the success icon from the DOM, so the success icon did no receive the click anymore which would have bubbled up to the cell.

Fixed by preventing the button's blur event in the `onMouseDown` callback on the success-icon. When releasing the mouse, it triggers the cell's onClick which takes care of getting into the editable state.